eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,23 +1,15 @@
+<!-- Whenever altering this or other Source Build files, please include @dotnet/source-build-internal as a reviewer. -->
```

Comment:
@oleksandr-didyk - I am proposing a slight wording change to reduce the length of the message.  I also introduced a blank link before the first element.